libs/icons/src/DeleteBinWithBg.tsx

```diff
@@ -0,0 +1,27 @@
+import { createIcon } from './create-icon';
+import { IconBase } from './types';
+
+export const DeleteBinWithBg = (props: IconBase) => {
+  return createIcon({
+    ...props,
+    path: [
+      <svg viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The `createIcon` function already creates an `<svg></svg>` wrapper, so that you won't need the svg wrapper here.